test/utils/hex.hpp

```diff
@@ -28,3 +28,17 @@ inline std::string hex(bytes_view data)
     return hex(data.data(), data.size());
 }
 }  // namespace fizzy
+
+/// Operator for 0x integer literals, e.g. 0x0a0b0c0d_b.
+inline fizzy::bytes operator""_b(const char* literal)
+{
+    if (literal[0] != '0' || literal[1] != 'x')
+        throw std::invalid_argument{"Only 0x literals are supported"};
+
+    return fizzy::from_hex(&literal[2]);
+}
+
+inline fizzy::bytes operator""_b(const char* literal, size_t /*length*/)
```

Comment:
This is for `""` literals. It might be confusing that hex digits are expected as the content, so this might be renamed to `_hb` or something else.

Reply:
I suggest we go for a single form only. And perhaps the `_hex` postfix is nicer?